### ISSUE TITLE
[Bug] 0 words, 0 characters when previewing tabs

### ISSUE THREAD

**Issue by danieltomasz** [Zettlr/Zettlr#793]
## Description

When previewing tabs they display
```
0 words, 0 characters 
```

even when they have content. I noticed this since 1.7.beta4

After writing on them and saving they display words and characters properly.

After restart the wordcount again disappear.

## Reproducing

Open notes in tabs [they show 0 count]
Edit & save them [they show count properly]
Restart Zettlr [they show 0 count again]

## Expected behaviour

**Screenshots**

## Platform
 - OS and version: MacOS 
 - Zettlr Version:1.7.beta4
 - Screen Resolution:

**Comment by nathanlesage** [comment]
It's there since the first tabs implementation was in and I have not been able to pinpoint this issue. Because as soon as you edit the file it somehow works and I have literally no idea what is happening. I've been poking around the source code for hours already …

**Comment by danieltomasz** [comment]
> It's there since the first tabs implementation was in and I have not been able to pinpoint this issue. Because as soon as you edit the file it somehow works and I have literally no idea what is happening. I've been poking around the source code for hours already …

@nathanlesage I dont know if this  matter but you have two different Case sensitive variables  in file `zettlr-renderer.js`

```
  file.wordcount = this._editor.getWrittenWords() 
```
in line 563

and then in function `refreshCurrentFile`  in 359

```
f.wordCount = file.wordCount

```

and you use the `file.wordcount`only in one place
```
./source/main/commands/file-save.js:38:    this._app.stats.updateWordCount(file.wordcount || 0)
```

**Comment by nathanlesage** [comment]
Good eye! I should definitely rename that. Nevertheless, it was not the cause :/

**Comment by nathanlesage** [comment]
I have literally NO IDEA when it happened, but I just realized that suddenly it works o.O

I currently assume it has something to do with the newly added `localizeNumber`, but I can't really explain it …

Feel free to comment if I break it again :D

**Comment by danieltomasz** [comment]
On my machine, when I build your latest commit I am still getting this  0 words error, let's wait for others what they get ...

**Comment by nathanlesage** [comment]
Ah dammit :(

**Comment by danieltomasz** [comment]
> Ah dammit :(

When debugging in the VSC I could see the proper word count, but not when using the yarn build

**Comment by nathanlesage** [comment]
> When debugging in the VSC I could see the proper word count

Yeah, I assumed that it's fixed on develop (albeit not on builds, because, well, I'm not too fond of the frequency of beta builds atm :D)

**Comment by nathanlesage** [comment]
So for me it's working now, what about you?

**Comment by nathanlesage** [comment]
Found out what it was: At some point I apparently accidentally introduced an error in the cache so that the cache saved `0` as wordCount, but not the real one. I've just committed another patch that *should* fix that. What will surely help is to clear the cache (simply navigate to the application data folder, e.g. `/Users/name/Application support/zettlr/fsal/cache` on macOS and remove all the files with names from `-1` to `99`)
